src/main/java/racingcar/model/RacingGame.java

```diff
@@ -0,0 +1,40 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import racingcar.domain.Car;
+import racingcar.dto.FinalResultDto;
+import racingcar.dto.RoundResultDto;
+
+public class RacingGame {
+
+    private final CarMover carMover;
+    private final List<RoundResultDto> roundResultList = new ArrayList<>();
+
+    public RacingGame(CarMover carMover) {
+        this.carMover = carMover;
+    }
+
+    public FinalResultDto playRacingGame(int roundNumber, List<Car> carList) {
+        for (int i = 0; i < roundNumber; i++) {
+            carMover.moveCars(carList);
+            roundResultList.add(new RoundResultDto(carList));
+        }
+
+        return new FinalResultDto(roundResultList, determineWinner(carList, getMaxPosition(carList)));
```

Comment:
이건 고려안하셔도 되지만 가독성을 높이기 위해 분리하면 좋을 거 같습니다!
```suggestion
        List<String> winnerList = determineWinner(carList, getMaxPosition(carList));
        return new FinalResultDto(roundResultList, winnerList);
```

Reply:
다시 사용하지 않을 변수들을 굳이 선언하지 않고 싶어서 바로 return했던 것인데, 가독성 면에서 오히려 안 좋을 수도 있겠네요